enos/modules/build_local/scripts/build.sh

```diff
@@ -11,5 +11,10 @@ export CGO_ENABLED=0
 
 root_dir="$(git rev-parse --show-toplevel)"
 pushd "$root_dir" > /dev/null
-make ci-build-ui ci-build ci-bundle
+make ci-build-ui ci-build
+
+: "${BUNDLE_PATH:=$(git rev-parse --show-toplevel)/vault.zip}"
```

Comment:
I wasn't familiar with this use of leading colon.  I read up on it and it sounds like the intent is to avoid emitting errors when the right hand side isn't a valid command, but that doesn't seem like it applies here.  Moreover we don't expect BUNDLE_PATH to ever be set before this line, do we?  In which case I think the more natural way to write this would be 

```
BUNDLE_PATH=$(git rev-parse --show-toplevel)/vault.zip
```

Reply:
I saw it being set [here](https://github.com/hashicorp/vault/blob/5ebda5d8f456aed8822d89f1f6d2bfaddf582bea/enos/modules/build_local/main.tf#L64) but wasn't familiar enough with the local enos flow to know when this was being set - I just copied it from where it used to be : https://github.com/hashicorp/vault/blob/5ebda5d8f456aed8822d89f1f6d2bfaddf582bea/scripts/ci-helper.sh#L193 as we don't need this in the CI builds anymore since the actions-go-build action does the bundling for us

Reply:
I'm responsible for the leading colon in the ci-helper. It's essentially a way to declare and initialize a variable unless it's already set. This make it easy for us to override any of the variables that are declared if we want to. In this case I don't see us overriding that environment variable so it's probably not necessary.

Reply:
On second thought, this is needed as we will pass the `BUNDLE_PATH` in from the outside and should respect  it https://github.com/hashicorp/vault/blob/6b329195388c73859f1860bcaf4092e11b1edd1e/enos/modules/build_local/main.tf#L64